docs/apache-airflow-providers-fab/auth-manager/access-control.rst

```diff
@@ -196,19 +200,28 @@ Website Permissions
 Action                                 Permissions                                                             Minimum Role
 ====================================== ======================================================================= ============
 Access homepage                        Website.can_read                                                        Viewer
+Show Browse menu                       Browse.menu_access                                                      Viewer
+Show DAGs menu                         DAGs.menu_access                                                        Viewer
 Get DAG stats                          DAGs.can_read, DAG Runs.can_read                                        Viewer
+Show Task Instances menu               Task Instances.menu_access                                              Viewer
 Get Task stats                         DAGs.can_read, DAG Runs.can_read, Task Instances.can_read               Viewer
 Get last DAG runs                      DAGs.can_read, DAG Runs.can_read                                        Viewer
 Get DAG code                           DAGs.can_read, DAG Code.can_read                                        Viewer
 Get DAG details                        DAGs.can_read, DAG Runs.can_read                                        Viewer
+Show DAG Dependencies menu             DAG Dependencies menu                                                   Viewer
+Get DAG Dependencies                   DAG Dependencies.can_read                                               Viewer
 Get rendered DAG                       DAGs.can_read, Task Instances.can_read                                  Viewer
 Get Logs with metadata                 DAGs.can_read, Task Instances.can_read, Task Logs.can_read              Viewer
 Get Log                                DAGs.can_read, Task Instances.can_read, Task Logs.can_read              Viewer
 Redirect to external Log               DAGs.can_read, Task Instances.can_read, Task Logs.can_read              Viewer
 Get Task                               DAGs.can_read, Task Instances.can_read                                  Viewer
+Show XCom menu                         XComs.menu_access                                                       Op
 Get XCom                               DAGs.can_read, Task Instances.can_read, XComs.can_read                  Viewer
+Create XCom                            XComs.can_create                                                        Op #TODO
```

Comment:
whats the TODO here?

Reply:
removed. this was unintended. thanks for pointing out